### PR TITLE
Update the H273 URL to something that exists.

### PR DESCRIPTION
**Pull request by vrabaud**
I believe the `S` is for `superseded`.

### CODE REVIEW

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks.

If the formulas are not renumbered in new versions, ww can also point to the top-level URL https://www.itu.int/rec/T-REC-H.273, which should be stable. But I don't know if these standards aim to keep the section numbers and formula numbers stable.

File: src/colr.c [hunk 1]
```diff
@@ -108,7 +108,7 @@ struct avifMatrixCoefficientsTable
     const float kb;
 };
 
-// https://www.itu.int/rec/T-REC-H.273-201612-I/en
+// https://www.itu.int/rec/T-REC-H.273-201612-S
```
Comment (wantehchang) [inline src/colr.c:111]: I am curious: in France is this URL redirected to https://www.itu.int/rec/T-REC-H.273-201612-S/fr?

Comment (vrabaud) [reply src/colr.c:111]: yes it does (from my personal account)

**Comment by vrabaud** [comment]
I did not want to check the most recent version (especially because there can be more in the future ...)

**Pull request merged**